### ISSUE TITLE
Please add support for Arabic (All dialects)

### ISSUE THREAD

**Issue by neoOpus** [myshell-ai/MeloTTS#101]
I find this text-to-speech technology to be truly incredible and I sincerely hope that it will soon include support for the Arabic language in all its various dialects.

**Comment by MuhammadAli-208** [comment]
Have u implemented any custom solution for Arabic support @neoOpus ?

**Comment by neoOpus** [comment]
@MuhammadAli-208 No, due to lack of financial support, I didn’t pursue this; unfortunately, Arabs neither value their language nor prioritize self-preservation, choosing instead to spend on, futilities, luxuries and leisure.

**Comment by m15kh** [comment]
@neoOpus 
Hi dude, have you found any solution for Arabic TTS with various accents? If yes, could you share how you did it and which repo you used?
